packages/sdk/src/types/Redemptions.ts

```diff
@@ -0,0 +1,123 @@
+import { OrdersItem, OrdersCreateResponse } from './Orders'
+import { SimpleCustomer } from './Customers'
+import { VouchersResponse } from './Vouchers'
+
+export interface RedemptionsRedeemBody {
+	tracking_id?: string
+	customer?: SimpleCustomer & { description?: string }
+	order?: Pick<OrdersCreateResponse, 'id' | 'source_id' | 'amount' | 'items' | 'status' | 'metadata'>
+	metadata?: Record<string, any>
+	reward?: {
+		id?: string
+		points?: number
+	}
+	gift?: {
+		credits: number
+	}
+	session?: {
+		key: string
+	}
+}
+
+export interface RedemptionsRedeemResponse {
+	id: string
+	object: 'redemption'
+	date?: string
+	customer_id?: string
+	tracking_id?: string
+	order?: OrdersCreateResponse
+	metadata?: Record<string, any>
+	result?: string
+	voucher?: VouchersResponse
+}
+
+export interface RedemptionsListParams {
+	limit?: number
+	page?: number
+	result?: 'SUCCESS' | 'FAILURE'
+	campaign?: string
+	customer?: string
+	created_at?: {
+		before?: string
+		after?: string
+	}
+}
+
+export interface RedemptionsListResponse {
+	object: 'list'
+	total: number
+	data_ref: 'redemptions'
+	redemptions: {
```

Comment:
Definition of single redemption is missing fields:
```
"failure_code"?:string,
"failure_message"?:string
```
Those fields are returned when redemption has result: `FAILURE`

Example:
```
"failure_code":"missing_customer",
"failure_message":"Missing Customer"
```